switchbot/const/const.py

```diff
@@ -0,0 +1,150 @@
+COMMAND_SET_BRIGHTNESS = {
+    SwitchbotModel.CEILING_LIGHT: f"{CEILING_LIGHT_CONTROL_HEADER}01FF01{{}}",  # brightness, color_temp
+    SwitchbotModel.COLOR_BULB: f"{COLOR_BULB_CONTROL_HEADER}14{{}}",  # brightness
+    SwitchbotModel.FLOOR_LAMP: f"{LIGHT_STRIP_CONTROL_HEADER}14{{}}",
+    SwitchbotModel.LIGHT_STRIP: f"{LIGHT_STRIP_CONTROL_HEADER}14{{}}",
+    SwitchbotModel.STRIP_LIGHT_3: f"{LIGHT_STRIP_CONTROL_HEADER}14{{}}",
+}
+COMMAND_SET_COLOR_TEMP = {
+    SwitchbotModel.CEILING_LIGHT: f"{CEILING_LIGHT_CONTROL_HEADER}01FF01{{}}",  # brightness, color_temp
+    SwitchbotModel.COLOR_BULB: f"{COLOR_BULB_CONTROL_HEADER}13{{}}",  # color_temp
+    SwitchbotModel.FLOOR_LAMP: f"{LIGHT_STRIP_CONTROL_HEADER}11{{}}",
+    SwitchbotModel.STRIP_LIGHT_3: f"{LIGHT_STRIP_CONTROL_HEADER}11{{}}",
+}
+COMMAND_SET_RGB = {
+    SwitchbotModel.COLOR_BULB: f"{COLOR_BULB_CONTROL_HEADER}12{{}}",  # brightness, red, green, blue
+    SwitchbotModel.FLOOR_LAMP: f"{LIGHT_STRIP_CONTROL_HEADER}12{{}}",
+    SwitchbotModel.LIGHT_STRIP: f"{LIGHT_STRIP_CONTROL_HEADER}12{{}}",
+    SwitchbotModel.STRIP_LIGHT_3: f"{LIGHT_STRIP_CONTROL_HEADER}12{{}}",
+}
+COMMAND_DEVICE_GET_BASIC_INFO = {
+    SwitchbotModel.CEILING_LIGHT: ["5702", "570f5581"],  # version info, basic info
+    SwitchbotModel.COLOR_BULB: ["570003", "570f4801"],
+    SwitchbotModel.LIGHT_STRIP: [
+        "570003",
+        "570f4A01",
+    ],  # for light strip, floor lamp, and strip light 3
+}
+COMMAND_TURN_ON = {
+    SwitchbotModel.CEILING_LIGHT: f"{CEILING_LIGHT_CONTROL_HEADER}01FF01FFFF",
+    SwitchbotModel.COLOR_BULB: f"{COLOR_BULB_CONTROL_HEADER}01",
+    SwitchbotModel.FLOOR_LAMP: f"{LIGHT_STRIP_CONTROL_HEADER}01",
+    SwitchbotModel.LIGHT_STRIP: f"{LIGHT_STRIP_CONTROL_HEADER}01",
+    SwitchbotModel.STRIP_LIGHT_3: f"{LIGHT_STRIP_CONTROL_HEADER}01",
+    SwitchbotModel.AIR_PURIFIER: "570F4C010100",
+    SwitchbotModel.AIR_PURIFIER_TABLE: "570F4C010100",
+    SwitchbotModel.EVAPORATIVE_HUMIDIFIER: "570F430101",
+    SwitchbotModel.CIRCULATOR_FAN: "570F410101",
+    SwitchbotModel.RELAY_SWITCH_1: "570F70010100",
+    SwitchbotModel.RELAY_SWITCH_1PM: "570F70010100",
+    SwitchbotModel.GARAGE_DOOR_OPENER: "570F70010100",
+}
+COMMAND_TURN_OFF = {
+    SwitchbotModel.CEILING_LIGHT: f"{CEILING_LIGHT_CONTROL_HEADER}02FF01FFFF",
+    SwitchbotModel.COLOR_BULB: f"{COLOR_BULB_CONTROL_HEADER}02",
+    SwitchbotModel.FLOOR_LAMP: f"{LIGHT_STRIP_CONTROL_HEADER}02",
+    SwitchbotModel.LIGHT_STRIP: f"{LIGHT_STRIP_CONTROL_HEADER}02",
+    SwitchbotModel.STRIP_LIGHT_3: f"{LIGHT_STRIP_CONTROL_HEADER}02",
+    SwitchbotModel.AIR_PURIFIER: "570F4C010000",
+    SwitchbotModel.AIR_PURIFIER_TABLE: "570F4C010100",
+    SwitchbotModel.EVAPORATIVE_HUMIDIFIER: "570F430100",
+    SwitchbotModel.CIRCULATOR_FAN: "570F410102",
+    SwitchbotModel.RELAY_SWITCH_1: "570f70010000",
+    SwitchbotModel.RELAY_SWITCH_1PM: "570f70010000",
+    SwitchbotModel.GARAGE_DOOR_OPENER: "570f70010000",
+}
```

Comment:
It would make more sense to make these classvars on each class so you can do self._TURN_ON_COMMAND